src/platform/stub-setup.ts

```diff
@@ -5,31 +5,50 @@
 import {AppStub} from './smart-home-app';
 import {MockLocalHomePlatform} from './mock-local-home-platform';
 import {DeviceManagerStub} from './device-manager';
+import {ExecuteStub} from './execute';
+import {AssertionError} from 'assert';
 
 export const smarthomeStub: {
   App: typeof smarthome.App;
+  Execute: typeof smarthome.Execute;
   Intents: {[key in keyof typeof smarthome.Intents]: string};
   DataFlow: {
     UdpRequestData: typeof smarthome.DataFlow.UdpRequestData;
   };
+  IntentFlow: {
+    HandlerError: typeof smarthome.IntentFlow.HandlerError;
+  };
   Constants: {
     Protocol: {[key in keyof typeof smarthome.Constants.Protocol]: string};
   };
-  Execute: typeof smarthome.Execute;
 } = {
   App: AppStub,
+  Execute: ExecuteStub,
   Intents: {
     EXECUTE: 'action.devices.EXECUTE',
     IDENTIFY: 'action.devices.IDENTIFY',
     REACHABLE_DEVICES: 'action.devices.REACHABLE_DEVICES',
   },
+  IntentFlow: {
+    HandlerError: class extends Error {
+      constructor(requestId: string, errorCode?: string, debugString?: string) {
+        super(errorCode);
+        this.requestId = requestId;
+        this.errorCode = errorCode || ' ';
```

Comment:
Does this empty string need to have a space?

Reply:
No, it should be empty.  Fixed